tests/Tokenizer/TokenTest.php

```diff
@@ -285,24 +285,34 @@ public function provideIsWhitespaceCases()
         );
     }
 
-    public function testPropertiesOfArrayToken()
+    /**
+     * @param array|string|mixed $prototype
```

Comment:
I was wondering how to nicely write it, `array|string` is valid subset of types, while `mixed` is unvalid subset. together it's `array|string|mixed`.